wcmc_components/lib/wcmc_components/filterable/table_attributes.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# TableAttributes is a service object class that can store information about an ActiveModel class's "table attributes"
+
+class TableAttributes
+  attr_reader :attributes
```

Comment:
Intentionally omitting a setter for attributes. #add_attribute should be used instead